### ISSUE TITLE
Configuration?

### ISSUE THREAD

**Issue by smlombardi** [AtomLinter/linter-csslint#52]
As mentioned in https://github.com/AtomLinter/linter-csslint/issues/49  a `.csslintrc` file on the project root has no effect on the linter; you can disable a check and it is still flagged.

I have tried using the Atom config file; of course since there's no documentation for this I may have tried it wrong:
````
"linter-csslint": [
    {
      important: false
    }
  ]
````
I tried to follow the pattern for other linters. The above had no affect either.

The ONLY thing that seems to work is the [Embedded Rulesets](https://github.com/CSSLint/csslint/wiki/Command-line-interface#embedded-rulesets-new-in-v0910), as in add this to the css file:

`/*csslint empty-rules: false, important: false*/`

Please explain how to configure the rules, preferable via an .rc file so we can make our rules sharable with our team.

**Comment by Arcanemagus** [comment]
This package simply calls `csslint`, as such any standard method for configuring that should work equally well here. For an explination as to why it might not be working see https://github.com/AtomLinter/linter-csslint/issues/49#issuecomment-145619678.

I'm going to mark this issue as closed since it is essentially a duplicate of #49.
